types/dialogflow/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for dialogflow 0.9
+// Type definitions for dialogflow 1.0
 // Project: https://github.com/googleapis/nodejs-dialogflow
-// Definitions by: Daniel Dyla <https://github.com/dyladan>, Tom Carrio <https://github.com/tcarrio>
+// Definitions by: Daniel Dyla <https://github.com/dyladan>
+//                 Tom Carrio <https://github.com/tcarrio>
+//                 Khalil Choudhry <https://github.com/khalilchoudhry>
```

Comment:
This header is typically for maintainers, not just attribution. Did you mean to add yourself as a maintainer?

Reply:
Actually I thought it's for attribution, but now, I would like to be maintainer as I actively use dialogflow's nodejs client.

Reply:
ok no problem I approved the request